examples/cloudrun/wait-approval/.pipe.yaml

```diff
@@ -0,0 +1,26 @@
+# Deployment pipeline that contains a manual approval stage.
+apiVersion: pipecd.dev/v1beta1
+kind: CloudRunApp
+spec:
+  notification:
+    pipeline:
+    stages:
+      # Promote new version to receive amount of traffic.
+      - name: CLOUDRUN_PROMOTE
+        with:
+          percent: 10
+      - name: WAIT_APPOVAL
+      # Promote new version to receive amount of traffic.
+      - name: CLOUDRUN_PROMOTE
+        with:
+          percent: 50
+      - name: WAIT_APPOVAL
+      # Promote new version to receive all traffic.
+      - name: CLOUDRUN_PROMOTE
+        with:
+          percent: 100
+    mentions:
+      - event: DEPLOYMENT_WAIT_APPROVAL
+        slack:
+          - U024BE7LH # nghialv
+          - U025BE7LK # nakabonne
```

Comment:
Could you replace these by dummy data? For example,

``` yaml
- slack-user-id-1  # slack-user-name-1
- slack-user-id-2  # slack-user-name-2
```

Reply:
3ade9a1